tests/test_permissions.py

```diff
@@ -0,0 +1,1254 @@
+    ),
+    field_id('Mutation', 'createNetwork'): TestData(tenant_perm=MANAGE_ENVIRONMENTS, resource_perm=CREATE_NETWORK),
+    field_id('Mutation', 'createOmicsRun'): TestData(tenant_perm=MANAGE_OMICS_RUNS, resource_perm=CREATE_OMICS_RUN),
+    field_id('Mutation', 'createOrganization'): TestData(
+        tenant_perm=MANAGE_ORGANIZATIONS, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'createQuicksightDataSourceSet'): TestData(
+        tenant_ignore=IgnoreReason.TENANT, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'createRedshiftConnection'): TestData(
+        tenant_ignore=IgnoreReason.BACKPORT, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'createSagemakerNotebook'): TestData(
+        tenant_perm=MANAGE_NOTEBOOKS, resource_perm=CREATE_NOTEBOOK
+    ),
+    field_id('Mutation', 'createSagemakerStudioUser'): TestData(
+        tenant_perm=MANAGE_SGMSTUDIO_USERS, resource_perm=CREATE_SGMSTUDIO_USER
+    ),
+    field_id('Mutation', 'createShareObject'): TestData(tenant_perm=MANAGE_SHARES, resource_perm=CREATE_SHARE_OBJECT),
+    field_id('Mutation', 'createTableDataFilter'): TestData(
+        tenant_perm=MANAGE_DATASETS, resource_perm=CREATE_TABLE_DATA_FILTER
+    ),
+    field_id('Mutation', 'createTerm'): TestData(
+        tenant_perm=MANAGE_GLOSSARIES, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'createWorksheet'): TestData(
+        tenant_perm=MANAGE_WORKSHEETS, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteAttachedMetadataForm'): TestData(
+        tenant_ignore=IgnoreReason.BACKPORT, resource_perm=ATTACH_METADATA_FORM
+    ),
+    field_id('Mutation', 'deleteCategory'): TestData(
+        tenant_perm=MANAGE_GLOSSARIES, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteConnectionGroupPermission'): TestData(
+        tenant_ignore=IgnoreReason.BACKPORT, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteDashboard'): TestData(tenant_perm=MANAGE_DASHBOARDS, resource_perm=DELETE_DASHBOARD),
+    field_id('Mutation', 'deleteDataPipeline'): TestData(tenant_perm=MANAGE_PIPELINES, resource_perm=DELETE_PIPELINE),
+    field_id('Mutation', 'deleteDataPipelineEnvironment'): TestData(
+        tenant_perm=MANAGE_PIPELINES, resource_perm=UPDATE_PIPELINE
+    ),
+    field_id('Mutation', 'deleteDataset'): TestData(tenant_perm=MANAGE_DATASETS, resource_perm=DELETE_DATASET),
+    field_id('Mutation', 'deleteDatasetStorageLocation'): TestData(
+        tenant_perm=MANAGE_DATASETS, resource_perm=DELETE_DATASET_FOLDER
+    ),
+    field_id('Mutation', 'deleteDatasetTable'): TestData(
+        tenant_perm=MANAGE_DATASETS, resource_perm=DELETE_DATASET_TABLE
+    ),
+    field_id('Mutation', 'deleteEnvironment'): TestData(
+        tenant_perm=MANAGE_ENVIRONMENTS, resource_perm=DELETE_ENVIRONMENT
+    ),
```

Comment:
All glossary mutations have CUSTOM resource permissions

Reply:
added is_owner check

Reply:
++ Term and Category Mutations too I think

Reply:
I have `glossary_owner_perm=True` in all Term/Category/Glossary mutations